tests/performance_benchmarks.py

```
@@ -0,0 +1,256 @@
+            
+            assert len(results) == len(outputs)
+            
+            print(f"\n{evaluator.name} Performance:")
+            print(f"  Time for {len(outputs)} outputs: {execution_time:.2f}s")
+            print(f"  Time per output: {execution_time/len(outputs):.3f}s")
+            
+            # Performance target
+            assert execution_time < 10  # Should evaluate 10 outputs in under 10s
+
+    @pytest.mark.asyncio
+    async def test_evolution_performance(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_evolution_performance`

Add return type annotation: `None`

(ANN201)